### ISSUE TITLE
Python 3.6 format string in a function

### ISSUE THREAD

**Issue by brandm** [donkirkby/live-py-plugin#99]
Hi Don

First thank you very much for your effort dedicated to this very useful project!

With the most recent live-py-plugin 8c9db32 from 2017-01-25 this live-py-output in Emacs is as expected:

```Python
# Python code:           # # live-py-output:
import sys               #
v = sys.version[:6]      # v = '3.6.0 '
g = "{}".format(v)       # g = '3.6.0 '
def func():              #
    l = "{}".format(v)   # l = '3.6.0 '
func()                   #
```

But here I expect the same live-py-output as above:

```Python
# Python code:           # # live-py-output:
import sys               #              |
v = sys.version[:6]      # v = '3.6.0 ' |
g = f"{v}"               # g = '3.6.0 ' |
def func():              #              |
    l = f"{v}"           #              | l = '3.6.0 '
func()                   #
```

Michael

**Comment by donkirkby** [comment]
Thanks for the clear report, @brandm, I'll check it out. I've been meaning to add support for Python 3.6, but I haven't gotten around to it yet. Now's the time, it seems.

By the way, have you tried the new features in the Emacs mode for changing the Python version or using a unit test to drive your live coding?

**Comment by brandm** [comment]
Answer 1: As I currently stick to a single Python version and do this with `virtualenv` I don't use your new Emacs feature `live-py-set-version` yet.

Answer 2: Your new Emacs feature `live-py-set-driver` looks very promising for later when I want to use a unit test suite with simultaneous live coding. For the code I wrote until now I did "only" live coding and was so happy and felt so confident with it that I postponed automatic unit tests and their expected results.

Regarding your new features bound to `C-c d`, `C-c w`, `C-c p` and `C-c v` in `live-py-mode-map`: I disabled them in my Emacs setup because they got in my way (https://github.com/brandm/emacs.d/commit/086ba24). `C-c a` through `C-c z` are reserved to the user. Sorry to only criticize here, I don't have an idea how other minor modes choose their keys and can not make a suggestion for `live-py-mode`.
